src/openrct2/entity/Guest.cpp

```diff
@@ -6857,13 +6860,13 @@ void Guest::UpdateSpriteType()
         return;
     }
 
-    if (Energy <= 64 && Happiness < 128)
+    if (Energy <= 64 && Energy > 0 && Happiness < 128)
     {
         SetSpriteType(PeepSpriteType::HeadDown);
         return;
     }
 
-    if (Energy <= 80 && Happiness < 128)
+    if (Energy <= 80 && Energy > 0 && Happiness < 128)
```

Comment:
I don't get why you wouldn't want to update their sprite, wouldn't this technically limit your ability of making scenes?

Reply:
> I don't get why you wouldn't want to update their sprite, wouldn't this technically limit your ability of making scenes?

When I froze a guest in a frame I like, it would change into the guest with his head down after a second or so, I would prefer to trigger sprite animations by hand.

Reply:
But doesn't it sound a bit inconsitent that it would not have the same exceptions for nauseous, for example? I guess it would be better to have no exceptions and then tinker with the guest stats so it gets in the sprite position you want

Reply:
Ah, I see what you mean. when its energy level is 32 and his head is down. Freezing it will get is head back up. 

I want to freeze it in place, without changing its sprite, it needs to stay in the position it's at.